data/src/main/java/com/yapp/itemfinder/data/network/api/lockerlist/LockerListApi.kt

```diff
@@ -0,0 +1,10 @@
+package com.yapp.itemfinder.data.network.api.lockerlist
+
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface LockerListApi {
+    @GET("containers/by-space-id/{spaceId}")
+    suspend fun getLockersBySpaceId(@Path("spaceId") spaceId: Long): Response<List<LockerResponse>>
```

Comment:
`p2;` Default ErrorHandler에 위임하기 위해서 List<LockerResponse>로 바꿔주세요
```suggestion
    suspend fun getLockersBySpaceId(@Path("spaceId") spaceId: Long): List<LockerResponse>
```

Reply:
수정했습니다!